Gateway/Http/Client/TransactionPosCloudSync.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2018 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+
+namespace Adyen\Payment\Gateway\Http\Client;
+
+use Adyen\Util\Util;
+use Magento\Payment\Gateway\Http\ClientInterface;
+
+class TransactionPosCloudSync implements ClientInterface
+{
+    /**
+     * @var int
+     */
+    protected $storeId;
+
+	/**
+	 * @var \Adyen\Client
+	 */
+    protected $client;
+
+	/**
+	 * @var \Adyen\Payment\Helper\Data
+	 */
+    protected $adyenHelper;
+
+	/**
+	 * @var \Adyen\Payment\Logger\AdyenLogger
+	 */
+    protected $adyenLogger;
+
+    public function __construct(
+        \Adyen\Payment\Helper\Data $adyenHelper,
+        \Adyen\Payment\Logger\AdyenLogger $adyenLogger,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        array $data = []
+    ) {
+        $this->adyenHelper = $adyenHelper;
+        $this->adyenLogger = $adyenLogger;
+        $this->storeId = $storeManager->getStore()->getId();
+
+        // initialize client
+        $client = $this->adyenHelper->initializeAdyenClient($this->storeId);
+        $apiKey = $this->adyenHelper->getPosApiKey($this->storeId);
+        $client->setXApiKey($apiKey);
```

Comment:
```suggestion
        $apiKey = $this->adyenHelper->getPosApiKey($this->storeId);
        $client = $this->adyenHelper->initializeAdyenClient($this->storeId, $apiKey);
```

It looks like I missed it last time